pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -350,7 +351,12 @@ public Representation post(Representation entity) {
 
           // 3/ Request is parsed by the handler which generates a
           // list of FileItems
-          items = upload.parseRequest(getRequest());
+          try {
+            items = upload.parseRequest(getRequest());
+          } catch (FileUploadBase.IOFileUploadException e) {
+            LOGGER.info("Exception uploading segment: " + e);
```

Comment:
Discussed this offline, and it looks like the way API handles it, it could be a local error or a remote error, we cannot say. So, this should remain as an error with the stack, I suppose.